### ISSUE TITLE
Still maintained?

### ISSUE THREAD

**Issue by yujinzheng** [mybatis/generator#813]
Will you continue to maintain this software in the future? I notice you haven't released a new release for over two years.

**Comment by jeffgbutler** [comment]
Yes. There have been quite a few updates to the generator since the last release and you can read about them here: https://github.com/mybatis/generator/blob/master/core/mybatis-generator-core/src/site/xhtml/whatsNew.xhtml (or review the commit history).

The most significant changes are related to Kotlin code generation - which is still relatively new so it makes sense there is movement there.

The generator is fairly stable for Java code generation.

**Comment by mattiapitossi** [comment]
I also have some compatibility problems with Java 17 and Spring. In particular, maven cannot find the dependency, 1.4.1 is not officially available.

**Comment by jeffgbutler** [comment]
1.4.1 is not released yet, but the snapshot version is available.

If you have issues with Java 17, please open a new issue to let us know the details.
